src/dataset-builder/DemographicsChart.ts

```diff
@@ -7,7 +7,8 @@ interface ChartLabel {
 }
 export interface ChartSeries {
   name?: string;
-  data: number[];
+  colorByPoint?: boolean;
+  data: any /* number[] | number[][] */;
```

Comment:
This is a temporary fix to deal with build failures due to a typing error in the function `addToSeriesData` in this file, specifically in the line `series[i].data.push(data[i])` with `data[i]` erroring - Originally data was only ever of the type `number[]`, but for the column charts, I needed to use data of the form `number[][]`, hence the (commented) change. This caused errors, so I'm making it `any` for now, while I work on a solution